packages/ethpm-v1/README.md

```diff
@@ -0,0 +1,9 @@
+# `ethpm-v1`
+
+## Usage
+
+```
+const ethpmV1 = require('ethpm-v1');
+```
+
+This package contains support for the deprecated v1 version of the ethpm specification for backwards compatibility purposes.
```

Comment:
Just as a heads-up, I haven't done any editing / testing to the v1 code. It was just copied into its own package from the initial implementation inside the core package. The v1 tests were (and still are) set to be skipped.